Fix Lithuanian fork link.

ar1ja opened the pull request:
Migrated to: <https://git.ari.lt/ari/rudys> (last year already :') + I changed my username on GitHub anyway)

(yes sorry a fork already exists in my account and therefore I just put it on the @ari-lt org horses are gonna catch me for this deed)

ar1ja committed:
Fix Lithuanian fork link.

Migrated to: <https://git.ari.lt/ari/rudys> (last year already :') + I changed my username on GitHub anyway)

bnjbvr reviewed (commented):
I don't endorse the swearing on README, so closing; i'll remove the original link too.

ar1ja commented:
no worries, i just played into the meme and it wasnt an issue 2 yrs ago so thought its all fine

On 2 June 2025 10:44:08 EEST, Benjamin Bouvier ***@***.***> wrote:
>@bnjbvr commented on this pull request.
>
>I don't endorse the swearing on README, so closing; i'll remove the original link too.
>
>
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/bnjbvr/rouille/pull/81#pullrequestreview-2887160417
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
Ms. Arija A. / Ari Archer <https://ari.lt/>
(this e-mail was sent from a phone - cannot sign)

ar1ja commented:
> Migrated to: https://git.ari.lt/ari/rudys (last year already :') + I changed my username on GitHub anyway)
> 
> (yes sorry a fork already exists in my account and therefore I just put it on the @ari-lt org horses are gonna catch me for this deed)

ok few things

1. in my defence the license is literally 'do what the fuck you want license' and i didnt use any cuss words outside of 'fuck' and 'rust' in my readme :nerd_face: 
2. i fixed it https://git.ari.lt/ari/rudys/commit/850ad99c64f425f2153bdb4d5206a7906502dd9b

anyway :D

bnjbvr commented:
Thou shalt be nice to Rust 😇

Lun 02 juin 2025, à 12:03, ar1ja a écrit :
> *ar1ja* left a comment (bnjbvr/rouille#81) <https://github.com/bnjbvr/rouille/pull/81#issuecomment-2929798854>
>> Migrated to: https://git.ari.lt/ari/rudys (last year already :') + I changed my username on GitHub anyway)
>> 
>> (yes sorry a fork already exists in my account and therefore I just put it on the @ari-lt <https://github.com/ari-lt> org horses are gonna catch me for this deed)
>> 
> ok few things
> 
>  1. in my defence the license is literally 'do what the fuck you want license' and i didnt use any cuss words outside of 'fuck' and 'rust' in my readme 🤓
>  2. i fixed it https://git.ari.lt/ari/rudys/commit/850ad99c64f425f2153bdb4d5206a7906502dd9b
> anyway :D
> 
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/bnjbvr/rouille/pull/81#issuecomment-2929798854>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAJAC47E2IHOVE7P3VZFJYD3BQOP7AVCNFSM6AAAAAB6CDX5XOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSMRZG44TQOBVGQ>.
> You are receiving this because you were mentioned.Message ID: ***@***.***>
>

ar1ja commented:
> Thou shalt be nice to Rust 😇 Lun 02 juin 2025, à 12:03, ar1ja a écrit :
> […](#)
> *ar1ja* left a comment ([bnjbvr/rouille#81](https://github.com/bnjbvr/rouille/pull/81)) <[#81 (comment)](https://github.com/bnjbvr/rouille/pull/81#issuecomment-2929798854)> > Migrated to: https://git.ari.lt/ari/rudys (last year already :') + I changed my username on GitHub anyway) > > (yes sorry a fork already exists in my account and therefore I just put it on the @ari-lt <https://github.com/ari-lt> org horses are gonna catch me for this deed) > ok few things 1. in my defence the license is literally 'do what the fuck you want license' and i didnt use any cuss words outside of 'fuck' and 'rust' in my readme 🤓 2. i fixed it https://git.ari.lt/ari/rudys/commit/850ad99c64f425f2153bdb4d5206a7906502dd9b anyway :D — Reply to this email directly, view it on GitHub <[#81 (comment)](https://github.com/bnjbvr/rouille/pull/81#issuecomment-2929798854)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAJAC47E2IHOVE7P3VZFJYD3BQOP7AVCNFSM6AAAAAB6CDX5XOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSMRZG44TQOBVGQ>. You are receiving this because you were mentioned.Message ID: ***@***.***>

smh my head :roll_eyes: rust is not [mentioned in the bible](https://github.com/jjj333-p/root-bible)

bnjbvr commented:
Feel free to rebase with your updated link, now that it's only mentioning 🐴!

ar1ja commented:
2 lazy to do anything right now im about to bed </3 just finishing up 
some work ill get to it eventually ill let the fork horse around for now

Ms. Arija A. / Ari Archer <https://ari.lt/>
OpenPGP key ID (RSA4096): 4FAD63E936B305906A6C4894A50D5B4B599AF8A2
OpenPGP key ID (ECC): 20A6FBAC2E07A6256E5A435A841A60BFE0406195

On 6/2/25 23:23, Benjamin Bouvier wrote:
> *bnjbvr* left a comment (bnjbvr/rouille#81) 
> <https://github.com/bnjbvr/rouille/pull/81#issuecomment-2932360634>
>
> Feel free to rebase with your updated link, now that it's only 
> mentioning 🐴!
>
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/bnjbvr/rouille/pull/81#issuecomment-2932360634>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/ARCLVBRMYPV2WU7MJE4YY6L3BSXFXAVCNFSM6AAAAAB6CDX5XOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSMZSGM3DANRTGQ>.
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>
>

ar1ja commented:
lmfao a person found my website through this pr that so funny just got an email from them linking here 😭
